### ISSUE TITLE
Sliders and Joints often jump to a different position or rotation when you stop grabbing them in a headless session

### ISSUE THREAD

**Issue by Nytra** [Yellow-Dog-Man/Resonite-Issues#399]
### Describe the bug?

If you have a Slider or Joint component being used in a headless session and the 'DontDrive' option has not been enabled on the component, the Slider or Joint will usually jump to a different position or rotation when you let go of it. This is bad and can be very frustrating when trying to be precise with these.

### To Reproduce

1) Join a headless session.
2) Make a Slider or Joint object (Ensure that the 'DontDrive' option is NOT ticked)
3) Start using the Slider or Joint by grabbing it
4) Observe that it often will jump to a different position or rotation when you let go of it

### Expected behavior

The Slider or Joint stays in the position or rotation it was at when you let go of it.

### Screenshots

N/a

### Resonite Version Number

Beta 2023.10.20.831

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Valve Index, Desktop

### Log Files

N/a

### Additional Context

https://github.com/Neos-Metaverse/NeosPublic/issues/3733
https://github.com/Neos-Metaverse/NeosPublic/issues/2185
https://github.com/Neos-Metaverse/NeosPublic/issues/1270

### Reporters

Nytra (Discord: nytra)

**Comment by troyBORG** [comment]
This needs to get fixed as its very annoying.

**Comment by Nytra** [comment]
Updating this with log files

[GRAPHICAL CLIENT - DESKTOP-H976HO2 - 2023.11.28.1365 - 2023-11-29 00_38_01.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13494842/GRAPHICAL.CLIENT.-.DESKTOP-H976HO2.-.2023.11.28.1365.-.2023-11-29.00_38_01.log)

[HEADLESS - DESKTOP-H976HO2 - 2023.11.28.1365 - 2023-11-29 00_37_47.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13494843/HEADLESS.-.DESKTOP-H976HO2.-.2023.11.28.1365.-.2023-11-29.00_37_47.log)

**Comment by troyBORG** [comment]
> Even if you believe there is nothing to be seen inside of it- please do not post bug reports without including log files, @Nytra.

They already know what the problem is and the solution, this was something from the old platform that was fixed with a mod a long time ago, and is still present in resonite.

I wish a lot of the bugs that mods had already fixed would have already been implemented a long time ago...

**Comment by Frooxius** [comment]
> > Even if you believe there is nothing to be seen inside of it- please do not post bug reports without including log files, @Nytra.
> 
> They already know what the problem is and the solution, this was something from the old platform that was fixed with a mod a long time ago, and is still present in resonite.
> 
> I wish a lot of the bugs that mods had already fixed would have already been implemented a long time ago...

We work as fast we can, but there is a lot of things to be dealt with, so please, have patience. I feel myself like I barely have time for anything other than work, stuff just piles up. Hearing stuff like that is a bit upsetting and hurtful.

**Comment by troyBORG** [comment]
>Hearing stuff like that is a bit upsetting and hurtful.

Sorry.  

I hope soon that things that mods are able to fix can just be done as a pull request instead  and just natively merged in.

**Comment by Nytra** [comment]
The way that mods do things is often very hacky and is more of a workaround for the problem instead of solving it at the root. Mods work by Harmony patching the existing game functions which is very different from native development. So it is really not as simple as just making a PR for mods to be added to the game.

**Comment by troyBORG** [comment]
Yeah but doesn't the mod somewhat know that hey changing this value is what fixes the issue so that it could told at least point them in the right direction so they know what to look for in the code.

**Comment by art0007i** [comment]
> Yeah but doesn't the mod somewhat know that hey changing this value is what fixes the issue so that it could, atleast point them in the right direction so they know what to look for in the code.

that's not always the case, most of the time there are many solutions to a problem, some worse and some better. mods usually take the path of least resistance which can be less optimal. additionally the team would have to spend time reading the mod code rather than trying to diagnose the issue themselves.
